### PR TITLE
sample outputs created  working with Kubernetes v1.24 without docker

### PR DESCRIPTION
**Pull request by SubhasmitaSw**
This PR resolves issue  #31423 
Sample Images produced using Kubernetes configured via Katakoda. 

Ps: This is my first PR, reviews requested. 
@celestehorgan @sftim

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -405,7 +406,7 @@ If you have pods that are impacted from the default 1 second timeout,
 you should update their probe timeout so that you're ready for the
 eventual removal of that feature gate.
 
-With the fix of the defect, for exec probes, on Kubernetes `1.20+` with the `dockershim` container runtime,
+With the fix of the defect, for exec probes, on Kubernetes `1.20+` with the `containerd` container runtime,
 the process inside the container may keep running even after probe returned failure because of the timeout.
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:410]: I'm not sure about this change. I don't know that the issue is present when you use containerd as the container runtime. I recommit omitting this update from the current PR, and then opening an new PR (for the _dev-1.24_ branch) that is specifically about this paragraph.

**Comment by sftim** [comment]
@SubhasmitaSw thanks for opening this.
Because of a concern, I don't yet recommend merging this PR.

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 1 above)
Comment (SubhasmitaSw) [reply content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:410]: 1. I'll have to update this PR unaltering line 408
2. A new PR (for the dev-1.24 branch) specific to this issue to be opened? 
is that what you meant @sftim?

Comment (shannonxtreme) [reply content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:410]: @SubhasmitaSw yup, unalter 408 and track it in a new PR or issue that targets that branch :)

Comment (SubhasmitaSw) [reply content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:410]: @shannonxtreme Thank you, I'll open a separate pr.

**Comment by reylejano** [comment]
Preview of updated page:
https://deploy-preview-31454--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

/lgtm

**Pull request merged**